napari_cellseg3d/plugin_dock.py

```diff
@@ -214,41 +263,40 @@ def update(self, slice_num):
             self.slice_num -= 1
             self.update_button()
 
-        self.time = datetime.now()
+        # self.time = datetime.now()
 
-        if self.button.text() == "Not checked":
-            self.time_label.setVisible(False)
-        else:
-            self.time_label.setVisible(True)
-            self.time_label.setText(
-                f"Previously completed in {self.df.at[self.df.index[self.slice_num], 'time']}"
-            )
+        # if self.button.text() == "Not checked":
+        #     self.time_label.setVisible(False)
+        # else:
+        #     self.time_label.setVisible(True)
+        #     self.time_label.setText(
+        #         f"Previously completed in {self.df.at[self.df.index[self.slice_num], 'time']}"
+        #     )
+        # self.df.at[0, "time"] = self.time_elapsed
 
     def button_func(self):  # updates csv every time you press button...
         if self.viewer.dims.ndisplay != 2:
             # TODO test if undefined behaviour or if okay
             warnings.warn("Please switch back to 2D mode !")
             return
-        if self.button.text() == "Not checked":
 
-            start_time = self.time
-            finish_time = datetime.now()
-            self.time = finish_time
+        self.update_time_csv()
+        # finish_time = datetime.now()
+        # self.time_elapsed = finish_time - self.start_time
 
-            time_diff = utils.time_difference(start_time, finish_time)
-            print(f"Time taken : {time_diff}")
+        if self.button.text() == "Not checked":
+            # time_diff = utils.time_difference(self.start_time, finish_time)
+            # print(f"Time taken : {time_diff}")
 
             self.button.setText("Checked")
-            self.time_label.setVisible(True)
-            self.time_label.setText(f"Completed in {time_diff}")
+            # self.time_label.setVisible(True)
+            # self.time_label.setText(f"Completed in {time_diff}")
```

Comment:
Overall looks great :) One minor comment is that one of the good things about version control is that you can always go back to previous commits to check modified code. Because of this, I would suggest to remove the commented parts here for better readability.

Reply:
Sure, sorry, I forgot to remove it once I was done. Fixed in the latest commit